remove some legacy garbage (#12436)
mpv-player/mpv: DOCS/man/input.rst

    (Before mpv 0.33.0, instead of returning -1, this property was unavailable
    if no playlist entry was current.)

>     Writing the current value will be ignored. Use the ``playlist-play-index``
>     command to restart the playback of the current entry if that is desired.

Reviewer: ```suggestion
    Writing the current value back to the property will have no effect.
    Use ``playlist-play-index`` to restart the playback of the current entry if desired.
```
